py/picca/bal_tools.py

```diff
@@ -0,0 +1,67 @@
+##Functions used for masking BAL features ##LE 2020
+
+
+import fitsio
+import scipy as sp
+
+def read_bal(bal_catalog): ##Based on read_dla from picca/py/picca/io.py
+    lst = ['THING_ID','VMIN_CIV_450','VMAX_CIV_450','VMIN_CIV_2000','VMAX_CIV_2000']
+    h = fitsio.FITS(bal_catalog)
+    bal_dict = { k :h[1][k][:] for k in lst }
```

Comment:
`bal_dict = {col: hdul[1][col][:] for col in columns_list}`

Also consider accessing the HDU using the HDU name. This would make it more robust to possible changes in the BAL catalogue format (say, for example, that BALs are included as an extension in the main quasar catalogue)

Reply:
Hi @LaurEnnesser, 
Have you thought about this? How many HDUs are there in the file? If there is only one and it is not named, then we can use the number, but I wanted to make sure you had considered this